config/prod/test-ubuntu.yaml

```diff
@@ -0,0 +1,21 @@
+# Provision an EC2 instance
+template_path: templates/managed-ec2.yaml
```

Comment:
`managed-ec2.yaml` has been deprecated.  Please change this to reference the new template version `managed-ec2-v2.yaml`